src/components/OwnerPanel/OwnerPanel.tsx

```diff
@@ -16,8 +17,8 @@ const OwnerPanel: React.SFC<Props> = ({ imageSrc, ownerName, className }) => {
       </figure>
       <h2 className={styles.ownerName}>{ownerName}</h2>
       <div className={styles.buttonContainer}>
-        <button className={classnames(styles.button, "button--filled")}>Donate</button>
-        <button className={classnames(styles.button, "button--outlined")}>Buy Gift Card</button>
+        <PopupModal merchant="Shunfa Bakery" option="Donation" className={classnames(styles.button, "button--filled")} />
```

Comment:
Is this going to change the text in the button from Donate to Donation? We should probably keep the button text at "Donate" and "Buy Gift Card".